frontend/src/components/Header.tsx

```diff
@@ -1,11 +1,15 @@
 import MyNavBar from "./MyNavBar";
+import {useContext} from "react";
+import AuthContext from "../context/authentication/AuthContext";
 
 export default function Header() {
+    const {userLoggedIn, logout} = useContext(AuthContext);
+
```

Comment:
Mhhh... warum hier den Hook und durchreichen anstatt ihn direkt in der NavBar aufzurufen?